### PR TITLE
fix: gatsby swag revival

### PR DESCRIPTION
**Pull request by Tejas-Sharma-dev**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

updated data.json for 'gatsby' swag, which was marked as 'expired' (ref: #1268)

- updated the swag link
- removed 'expired' tag

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -426,10 +426,10 @@
     "name": "Gatsby",
     "difficulty": "hard",
     "description": "Get a pull request merged into Gatsby and receive a free shirt, socks, or stickers.",
-    "reference": "https://www.gatsbyjs.org/docs/contributor-swag/",
+    "reference": "https://www.gatsbyjs.com/contributing/contributor-swag/",
     "image": "https://i.imgur.com/e7GwaTN.png",
     "dateAdded": "2018-10-24T11:42:09.000Z",
```
Comment (aslafy-z) [inline data.json:431]: Please update dateAdded to now please. We might want to rename or add another field for the updated date in the future.

File: data.json [hunk 2]
```diff
@@ -426,10 +426,10 @@
     "name": "Gatsby",
     "difficulty": "hard",
     "description": "Get a pull request merged into Gatsby and receive a free shirt, socks, or stickers.",
-    "reference": "https://www.gatsbyjs.org/docs/contributor-swag/",
+    "reference": "https://www.gatsbyjs.com/contributing/contributor-swag/",
```
Comment (aslafy-z) [inline data.json:429]: It looks like the actual swag is Netlify swag, not Gatsby one. I think we want to update the Netlify entry instead and keep that one expired.

Comment (Tejas-Sharma-dev) [reply data.json:429]: sounds good, why don't we update as 'Gatsby x Netlify' swag?

Comment (aslafy-z) [reply data.json:429]: It's up to you

Comment (Tejas-Sharma-dev) [reply data.json:429]: fine then, I'II just update the name and date. instead of adding new swag.

File: data.json [hunk 3]
```diff
@@ -423,13 +423,13 @@
     "tags": ["clothing", "hacktoberfest", "stickers", "accessories", "expired"]
   },
   {
-    "name": "Gatsby",
+    "name": "Gatsby x Netlify",
     "difficulty": "hard",
     "description": "Get a pull request merged into Gatsby and receive a free shirt, socks, or stickers.",
-    "reference": "https://www.gatsbyjs.org/docs/contributor-swag/",
+    "reference": "https://www.gatsbyjs.com/contributing/contributor-swag/",
     "image": "https://i.imgur.com/e7GwaTN.png",
```
Comment (aslafy-z) [inline data.json:430]: Can you please also update the image with one from the store to match the new swags

File: data.json [hunk 4]
```diff
@@ -423,13 +423,13 @@
...
-    "dateAdded": "2018-10-24T11:42:09.000Z",
-    "tags": ["clothing", "stickers", "expired"]
+    "dateAdded": "2024-10-04T16:08:09.000Z",
+    "tags": ["clothing", "stickers"]
```
Comment (aslafy-z) [inline data.json:432]:
```suggestion
    "tags": ["clothing", "mug", "stickers"]
```

File: data.json (hunk 3 above)
Comment (Tejas-Sharma-dev) [reply data.json:430]: ok, i'm updating image URL

**Comment by aslafy-z** [comment]
Thank you @Tejas-Sharma-dev!

**Pull request merged**